include/config/config_graphics.h

```diff
@@ -150,3 +150,13 @@
  * will be used instead.
  */
 #define DEFAULT_CULLING_RADIUS 300
+
+/**
+ * Eases the textured screen transitions to make them look smoother.
+ */
+#define EASE_IN_SCREEN_TRANSITIONS
+
+/**
+ * Makes use of the tex edge render mode to deliver smooth transition textures
+ */
+#define SHARP_TRANSITION_TEXTURES
```

Comment:
Additional opinions desirable for whether or not these should be defaults. It seems different enough that I want to say keep them off by default, but then again we have stuff like HD coins enabled by default so idk.